vip-plugins/vip-plugins.php

```diff
@@ -169,13 +169,13 @@ function wpcom_vip_plugin_action_links( $actions, $plugin_file, $plugin_data, $c
 function wpcom_vip_option_active_plugins( $value, $option ) {
 	$code_plugins = wpcom_vip_get_filtered_loaded_plugins();
 
-	if ( ! empty( $value ) && ! empty( $code_plugins ) ) {
-		$value = array_merge( $value, $code_plugins );
+	if ( empty( $value ) ) {
```

Comment:
This check does not feel right. It `$value` is empty, then create empty array. I believe that what situations is being solved here is whether `$value` is or is not an array. The check, for readability of the code, should if `if ( false === is_array( $value ) )`